botwo/botwo.py

```diff
@@ -0,0 +1,188 @@
+import fnmatch
+
+SPECIAL_METHODS = {"get_paginator", "delete_objects"}
+COPY_METHODS = {"copy", "copy_object", "copy_upload_part"}
+
+
+def client(client_mapping, config):
+    botwo = Botwo()
+    return botwo.create_client_class(client_mapping, config)
+
+
+class Paginatwo(object):
```

Comment:
According to [PEP-8](https://www.python.org/dev/peps/pep-0008/#documentation-strings), this class and every user-visible method should have a docstring.

Reply:
Done